front/src/modules/rollingStock/components/RollingStockCard/RollingStockCardDetail.tsx

```diff
@@ -167,11 +167,18 @@ export default function RollingStockCardDetail({
                 <td className="text-primary">{t('rollingResistanceB')}</td>
               </tr>
               <tr>
-                <td>
-                  {Math.floor(
+                <td
+                  title={
+                    rollingStock.rolling_resistance?.C
+                      ? ((rollingStock.rolling_resistance.C * 10000) / 10000).toString()
+                      : undefined
+                  }
+                >
+                  {floor(
```

Comment:
really puzzling, 10000 suggests 4 decimals, by `floor` limits to 2 decimals

Reply:
Yes, it useless to do it twice

Reply:
thanks.
does lodash `floor` do something different than your function stripDecimalDigits

Reply:
Yes, lodash floor have a precision parameter